tests/NLog.WindowsIdentity.Tests/ImpersonatingTargetWrapperTests.cs

```diff
@@ -1,35 +1,35 @@
-// 
+//
 // Copyright (c) 2004-2021 Jaroslaw Kowalski <jaak@jkowalski.net>, Kim Christensen, Julian Verdurmen
-// 
+//
 // All rights reserved.
-// 
-// Redistribution and use in source and binary forms, with or without 
-// modification, are permitted provided that the following conditions 
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
 // are met:
-// 
-// * Redistributions of source code must retain the above copyright notice, 
-//   this list of conditions and the following disclaimer. 
-// 
+//
+// * Redistributions of source code must retain the above copyright notice,
```

Comment:
I guess  you have applied some all-document-fixes. Can you revert the change to the license-text ?

Reply:
Manually resolved via notepad